dashboard/app/views/levels/_level_group.haml

```diff
@@ -0,0 +1,49 @@
+- app = 'level_group'
+- data = @level.properties
+:javascript
+  window.levelData = #{data.to_json};
+
+= render partial: 'levels/common_audio'
+
+- data["levels"].each_with_index do |level_name, index|
+  - level_blob = Level.find_by_name(level_name)
+  - @level = level_blob
+  - # Parse the array of the level resuts; see sample below in JavaScript.
+  - if @last_attempt
+    - last_attempt = JSON.parse(@last_attempt).each.select{|item| item["level_id"]==@level.id}.first["result"]
```

Comment:
I think you don't need the each (since the thing you are calling each on is already an Enumerable). And you can use find instead of select.first http://ruby-doc.org/core-2.3.0/Enumerable.html#method-i-find